src/walkabout/src/ir.rs

```diff
@@ -307,7 +307,7 @@ fn analyze_type(ty: &syn::Type) -> Result<Type> {
                 };
 
                 match &*segment_name {
-                    "bool" | "usize" | "u64" | "i64" | "char" | "String" | "PathBuf" => {
+                    "bool" | "usize" | "u64" | "i64" | "char" | "String" | "PathBuf" | "i32" => {
```

Comment:
what was this needed for?

Reply:
it's some static analysis path that analyzes our AST structures, and it needs to be able to recognize all the external types we use within them. I don't know the details beyond that, but the point is that this is necessary now that I've added an `i32` (for the ID strategy) to the CSR connector.